docker/docker.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# TODO: Allow this to be configured by metadata.
+readonly DOCKER_VERSION='18.06.0~ce~3-0~debian'
+
+function is_master() {
+  local role="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
+  if [[ "$role" == 'Master' ]] ; then
+    true
+  else
+    false
+  fi
+}
+
+function get_docker_gpg() {
+  # Hard code the GPG key to prevent malicious payloads. The key was originally
+  # fetched by running:
+  # curl -fsSL https://download.docker.com/linux/$(. /etc/os-release; echo
```

Comment:
you hard code debian below, just hard code it here

Reply:
Done.